libs/community/tests/integration_tests/graphs/test_neo4j.py

```diff
@@ -266,12 +266,17 @@ def test_neo4j_filtering_labels() -> None:
     # Remove all constraints
     graph.query("CALL apoc.schema.assert({}, {})")
     graph.query(
-        "CREATE (:`_Bloom_Scene_`)-[:_Bloom_HAS_SCENE_]->(:`_Bloom_Perspective_`)"
+        """
+        CREATE (:_Bloom_Scene_{property_a: 'a'})
+        -[:_Bloom_HAS_SCENE_{property_b: 'b'}]
```

Comment:
I think we want to add a space between the label/type and the property map for clarity (and I'm actually surprised this works).

```suggestion
        CREATE (:_Bloom_Scene_ {property_a: 'a'})
        -[:_Bloom_HAS_SCENE_ {property_b: 'b'}]
```